packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -1,14 +1,14 @@
 <template>
-  <TopNavList v-if="versionList">
+  <TopNavList v-if="props.gql.versions">
```

Comment:
Why not keep the versionList as a computed?
It would make the template look cleaner (for my taste)

Reply:
Ok done